client/me/action-remove/index.jsx

```diff
@@ -1,32 +1,23 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	classnames = require( 'classnames' ),
-	omit = require( 'lodash/omit' ),
-	classNames = require( 'classnames' );
-
-module.exports = React.createClass( {
+import React from 'react';
+import classNames from 'classnames';
+import omit from 'lodash/omit';
 
+export default React.createClass( {
 	displayName: 'ActionRemove',
 
-	getDefaultProps: function() {
-		return {
-			isSubmitting: false,
-			isPrimary: true
-		};
-	},
-
-	render: function() {
-		var buttonClasses = classNames( {
+	render() {
+		const buttonClasses = classNames( {
 			'action-remove': true
 		} );
 
 		return (
 			<button
 				title={ this.translate( 'Remove', { textOnly: true } ) }
 				{ ...omit( this.props, 'className' ) }
```

Comment:
If you're overriding the prop by specifying a separate value on the following line, you shouldn't need to omit `className`, i.e.:

```js
{ ...this.props }
```

Should work just as well.